v3-docs/docs/cli/index.md

```diff
@@ -217,210 +217,123 @@ METEOR_CLIENT_ENTRYPOINT=client/main.js METEOR_SERVER_ENTRYPOINT=server/main.js
-
-Npm packages included: `@babel/runtime`, `meteor-node-stubs`.
-
-Meteor packages included: `meteor`, `standard-minifier-css`, `standard-minifier-js`, `es5-shim`, `ecmascript`, `typescript`, `shell-server`,
-`static-html`, `webapp`, `server-render`, `hot-module-replacement`.
-
-
-## --package {package}
-
-The command `meteor create --package package-name` creates a new package. If used in an existing app, it will create a
-package in the `packages` directory. Check the [Meteor Guide](https://guide.meteor.com/writing-atmosphere-packages.html)
-for more information on how to get started writing packages.
-
-
-## --prototype {prototype}
-
-The command `meteor create --prototype app-name` creates a project with the prototype purpose packages (`autopublish`
-and `insecure`). If you use them, you can change your collections quickly and create prototype apps very quickly.
-However, these packages are not supposed to be used in production.
-
-For more information about security, you can read our [security checklist](https://guide.meteor.com/security.html#checklist).
-It can be used with other flags that create apps, such as `--react`, `blaze`, or `--typescript`.
-
-
-## --react {react}
-
-The command `meteor create --react app-name` creates a Meteor app with [React](https://react.dev/) and
-[MongoDB](https://www.mongodb.com/). It functions in the same way as if you don't use any flags. To create a complete
-app, including testing and deployment, follow the [React tutorial](https://react-tutorial.meteor.com/).
-
-Npm packages included: `@babel/runtime`, `meteor-node-stubs`, `react`, `react-dom`.
-
-Meteor packages included: `meteor-base`, `mobile-experience`, `mongo`, `reactive-var`, `standard-minifier-css`,
-`standard-minifier-js`, `es5-shim`, `ecmascript`, `typescript`, `shell-server`, `hot-module-replacement`, `static-html`,
-`react-meteor-data`.
+```bash
+meteor create [options] app-name
+```
 
+::: tip Default Behavior
+Without any flags, `meteor create` generates a React project.
+:::
 
+### Basic Options
 
-## --release {release}
+| Option | Description |
+|--------|-------------|
+| `--from <url>` | Clone a Meteor project from a URL |
+| `--example <name>` | Use a specific example template |
+| `--list` | Show list of available examples |
+| `--release <version>` | Specify Meteor version (e.g., `--release 2.8`) |
+| `--prototype` | Include `autopublish` and `insecure` packages for rapid prototyping (not for production) |
+
+### Application Types
+
+| Option | Description | Tutorial |
+|--------|-------------|----------|
+| `--react` | Create a React app (default) | [v3 React Tutorial](https://docs.meteor.com/tutorials/react/), [v2 React Tutorial](https://react-tutorial.meteor.com/) |
+| `--vue` | Vue 3 + Tailwind CSS + Vite | [v3 Vue Tutorial](https://docs.meteor.com/tutorials/vue/meteorjs3-vue3-vue-meteor-tracker.html), [v2 Vue Tutorial](https://vue3-tutorial.meteor.com/) |
```

Comment:
```suggestion
| `--vue` | Vue 3 + Tailwind CSS + Vite | [v3 Vue Tutorial](https://docs.meteor.com/tutorials/vue/meteorjs3-vue3-vue-meteor-tracker.html) |
```

Can we remove this link. vue2 is EOL and the link goes to the old vue3 tutorial.